include/gatekeeper_main.h

```diff
@@ -37,6 +37,15 @@
  */
 extern int gatekeeper_logtype;
 
+#define BLOCK_LOGTYPE RTE_LOGTYPE_USER1
+
+#define B_LOG(level, ...)			        		\
+	do {								\
+		rte_log_set_level(BLOCK_LOGTYPE, RTE_LOG_DEBUG);	\
+		rte_log_ratelimit(RTE_LOG_ ## level,			\
+		BLOCK_LOGTYPE, "GATEKEEPER: " __VA_ARGS__);		\
+	} while (0)
+
```

Comment:
The definition of `BLOCK_LOGTYPE` is going to the other patch. Once the other patch comes first, we can drop the macro `B_LOG()`.